x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformChainIT.java

```diff
@@ -188,7 +188,7 @@ private void testChainedTransforms(final int numTransforms) throws Exception {
             assertFalse(aliasExists(destWriteAlias));
 
             String transformConfig = createTransformConfig(sourceIndex, destIndex, destReadAlias, destWriteAlias);
-            assertAcknowledged(putTransform(transformId, transformConfig, true, RequestOptions.DEFAULT));
+            putTransform(transformId, transformConfig, true, RequestOptions.DEFAULT);
```

Comment:
It was moved into the `putTransform` method so that everyone calling `putTransform` will also call `assertAcknowledged` (which closes the inputstream), then `putTransform` became a `void` to match the other API calls